hxinsight-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-hxinsight-extension/alfresco-global.properties

```diff
@@ -11,7 +11,7 @@ hxi.auth.providers.hyland-experience.grant-type=client_credentials
 hxi.auth.providers.hyland-experience.client-name=acs-insight-connector
 hxi.auth.providers.hyland-experience.client-id=
 hxi.auth.providers.hyland-experience.client-secret=
-hxi.auth.providers.hyland-experience.scope=hxp.integrations
+hxi.auth.providers.hyland-experience.scope=hxp.integrations hxp environment_authorization
```

Comment:
can't tell whether that really mandatory but this is how I tested it, I also added `iam.jit-capture` in the scope